src/quo2/components/buttons/button.cljs

```diff
@@ -6,70 +6,125 @@
+                                                    :pressed  colors/primary-60
                                                     :disabled colors/primary-50}}
-                     :secondary {:icon-color       colors/primary-50
-                                 :label            {:style {:color colors/primary-50}}
-                                 :background-color {:default  colors/primary-50-opa-20
+           :secondary       {:icon-color           colors/primary-50
+                             :label                {:style    {:color colors/primary-50}}
+                             :background-color     {:default  colors/primary-50-opa-20
                                                     :pressed  colors/primary-50-opa-40
                                                     :disabled colors/primary-50-opa-20}}
-                     :grey      {:icon-color       colors/black
-                                 :label            {:style {:color colors/black}}
-                                 :background-color {:default  colors/neutral-20
+           :grey            {:icon-color           colors/neutral-100
+                             :icon-secondary-color colors/neutral-50
+                             :label                {:style    {:color colors/neutral-100}}
+                             :background-color     {:default  colors/neutral-10
+                                                    :pressed  colors/neutral-20
+                                                    :disabled colors/neutral-10}}
+           :dark-grey       {:icon-color           colors/neutral-100
+                             :icon-secondary-color colors/neutral-50
+                             :label                {:style    {:color colors/neutral-100}}
+                             :background-color     {:default  colors/neutral-20
                                                     :pressed  colors/neutral-30
                                                     :disabled colors/neutral-20}}
-                     :outline   {:icon-color   colors/neutral-50
-                                 :label        {:style {:color colors/black}}
-                                 :border-color {:default colors/neutral-30
-                                                :pressed colors/neutral-40}}
-                     :ghost     {:icon-color       colors/neutral-50
-                                 :label            {:style {:color colors/black}}
-                                 :background-color {:pr:pressedess colors/neutral-10}}
-                     :success   {:icon-color       colors/white
-                                 :label            {:style {:color colors/white}}
-                                 :background-color {:default colors/success-50
-                                                    :pressed colors/success-70
-                                                    :disabled colors/success-50}}
```

Comment:
Figma buttons component doesn't have a success variant, and also I didn't find any use of this variant accepted in [activity_center](https://github.com/status-im/status-mobile/blob/develop/src/status_im/ui/screens/activity_center/views.cljs).
@ilmotta Please can you confirm if the success button is used for that component, and please can you also provide the Figma link? Thank you